packages/modules/chargepoints/internal_openwb/chargepoint_module.py

```diff
@@ -46,8 +46,10 @@ def __init__(self, local_charge_point_num: int, serial_client: ModbusSerialClien
     def __factory(self, serial_client: ModbusSerialClient_) -> Tuple[CONNECTION_MODULES, evse.Evse]:
         meter_config = NamedTuple("MeterConfig", [('type', CONNECTION_MODULES), ('modbus_id', int)])
         meter_configuration_options = [
-            [meter_config(sdm.Sdm630, modbus_id=105), meter_config(b32.B32, modbus_id=201)],
-            [meter_config(sdm.Sdm630, modbus_id=106)]
+            [meter_config(sdm.Sdm630, modbus_id=5),
+             meter_config(sdm.Sdm630, modbus_id=105),
+             meter_config(b32.B32, modbus_id=201)],
+            [meter_config(sdm.Sdm630, modbus_id=6), meter_config(sdm.Sdm630, modbus_id=106)]
```

Comment:
Ich würde auch den B23 als LP2 der Duo unterstützen. Wer weiß, wann das mal benötigt wird.
```suggestion
            [meter_config(sdm.Sdm630, modbus_id=6),
             meter_config(sdm.Sdm630, modbus_id=106)
             meter_config(b23.B23, modbus_id=202)]
```